src/System.Security.Permissions/ref/System.Security.Permissions.cs

```diff
@@ -43,41 +42,39 @@ public sealed partial class ConfigurationPermission : System.Security.CodeAccess
         public bool IsUnrestricted() { throw null; }
         public override System.Security.SecurityElement ToXml() { throw null; }
         public override System.Security.IPermission Union(System.Security.IPermission target) { throw null; }
-    }    
-    [System.AttributeUsageAttribute((System.AttributeTargets)(32767), AllowMultiple = true, Inherited = false)]
+    }
+    [System.AttributeUsageAttribute(System.AttributeTargets.All, AllowMultiple=true, Inherited=false)]
     public sealed partial class ConfigurationPermissionAttribute : System.Security.Permissions.CodeAccessSecurityAttribute
     {
-        public ConfigurationPermissionAttribute(System.Security.Permissions.SecurityAction action) : base(default(System.Security.Permissions.SecurityAction)) { }
+        public ConfigurationPermissionAttribute(System.Security.Permissions.SecurityAction action) : base (default(System.Security.Permissions.SecurityAction)) { }
         public override System.Security.IPermission CreatePermission() { throw null; }
     }
 }
-
 namespace System.Data.Common
 {
     public abstract partial class DBDataPermission : System.Security.CodeAccessPermission, System.Security.Permissions.IUnrestrictedPermission
     {
         protected DBDataPermission() { }
-        protected DBDataPermission(System.Data.Common.DBDataPermission permission) { }
-        protected DBDataPermission(System.Data.Common.DBDataPermissionAttribute permissionAttribute) { }
+        protected DBDataPermission(System.Data.Common.DBDataPermission dataPermission) { }
+        protected DBDataPermission(System.Data.Common.DBDataPermissionAttribute attribute) { }
         protected DBDataPermission(System.Security.Permissions.PermissionState state) { }
-        protected DBDataPermission(System.Security.Permissions.PermissionState state, bool allowBlankPassword) { }
-        public bool AllowBlankPassword { get; set; }
+        protected DBDataPermission(System.Security.Permissions.PermissionState state, bool blankPassword) { }
```

Comment:
please review: `allowBlankPassword` -> `blankPassword`
discrepancy between source/ref - which is correct?

+ other parameter renames.